recipes/configure-logstash-kibana.rb

```diff
@@ -0,0 +1,32 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: configure-logstash-kibana
+
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+
+(private_es_hostname, es_attributes) = node[:opsworks][:layers]['elasticsearch'][:instances].first
+
+stack_name = stack_shortname
+elk_info = get_elk_info
+
+template '/etc/logstash/conf.d/logstash.conf' do
```

Comment:
maybe set an explicit owner on this?  I might not've been consistent but I like the idea of being clear who manages this. It probably defaults to "root".